grunt-tc

```diff
@@ -9,10 +9,9 @@ set -o errexit
 
 ################################################################################
 # Build the static asset files for the project
-# Files will now be found in Public folder
+# Files will now be found in the public folder
 ################################################################################
 
-npm cache clean
```

Comment:
I don't see why it should be, it's just a disk cache of the package at a specific version. Re-downloading every NPM package adds a good 3-4 minutes to every build. 

From https://docs.npmjs.com/cli/cache: 
> All data that passes through the cache is fully verified for integrity on both insertion and extraction. Cache corruption will either trigger an error, or signal [...] that the data must be refetched, which it will do automatically. For this reason, it should never be necessary to clear the cache for any reason other than reclaiming disk space